api/src/models/FedoraObject.ts

```diff
@@ -130,6 +130,18 @@ export class FedoraObject {
         );
     }
 
+    async modifyLicense(stream: string, licenseKey: string) {
+        const licenses = this.config.licenses;
+        const url = licenses[licenseKey];
+        const licenseXml = `
+            <METS:rightsMD xmlns:METS="http://www.loc.gov/METS/" ID="0">
+                <METS:mdRef xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" MDTYPE="OTHER" MIMETYPE="text/html" OTHERMDTYPE="HTML" xlink:href="${url}">
```

Comment:
It would probably be a good idea to call `xmlescape` on the URL in case it contains special characters like `&`. See [here](https://github.com/FalveyLibraryTechnology/VuDLPrepJS/blob/d3ef26cdf66a3d87dc249a80364ef0c7787da810/api/src/services/Fedora.ts#L380) for an example.

Reply:
Made change.